draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2883,7 +2889,125 @@ https://video.example.com/k/movie1/4/013.ts
+	</t>
+	<t>
+	Example use cases:
+		<list style="symbols">
+		<t>
+		Priority:  uCDN may have multiple trigger commands in “pending” and/or “active” mode. The trigger commands may potentially take a long time to execute, for example due to policy constraints, large number of content objects affected by it or other dCDN business logic.
+		uCDN may wish to prescribe the order in which dCDN picks up its trigger commands for execution from the “pending” queue, by indicating a relative priority of each trigger. The priority would affect selection of trigger commands specific to the requesting uCDN. dCDN may separately prioritize triggers from multiple uCDNs subject to its business logic. 
+
+		Multiple priority-related sub use cases exist: 
+		<list style="symbols">
+		<t> (i) uCDN needs to introduce urgent “purge” or “invalidate” trigger into an existing queue of trigger commands to correct wrong versions of content objects published by it</t>
+		<t> (ii) uCDN needs to indicate which content objects should be prepositioned, purged or invalidated first, for example prepositioning newer released content before prepositioning updates to existing catalog</t>
+		</list>
+		</t>
+
+		<t>
+		Pre-requisite:  In some cases uCDN may wish to indicate what trigger commands should be processed and completed before another trigger command is processed. 
+			For example, uCDN may want to rectify incorrectly published content by purging content objects and then prepositioning them again. In this case, uCDN may want the preposition trigger command to be processed only after the purge trigger command has been processed because concurrent processing of these triggers may cause the new version of these content objects to be purged. Alternatively, uCDN may wish to avoid processing of purge and invalidation triggers in parallel to long-running preposition triggers that could lead to race conditions, and condition execution of such purge or revalidation triggers upon completion or cancellation of the pre-positioning triggers. 
+		The prerequisite requirement implies that a previius trigger reaches one of the following states:   
+		<list style="symbols">
+			<t>(i) “complete” or “processed” for successful completion</t>
+			<t>(ii) “failed” for failed processing</t>
+			<t>(iii) “cancelled” for completion of cancellation</t>
+		</list>
+
+
+		</t>
+		<t>
+		Urgency: Separately from priority, uCDN may wish to indicate that dCDN should process a trigger command without delay. This requirement applies to CI/T Trigger commands only,. dCDN MUST process all CI/T Cancel Commands immediately at all times.  Urgency is separate and additional to priority, which indicates order in which trigger commands should be processed yet doesn’t prescribe how soon each command needs to be executed. 
```

Comment:
,. -> , as the
Last sentence could use a few small corrections for readability: "Urgency is can be specified in tandem with Priority, as Priority indicates the order in which trigger commands should be processed, yet does not prescribe how soon each command should be executed.

Reply:
Rewrote as:
                Urgency: uCDN may wish to indicate that dCDN should process a trigger command without delay.
                This requirement is separate and additional to priority, as priority indicates the order in which
                triggers should be processed, yet does not prescribe how soon each trigger should be executed.